Library/Formula/fish.rb

```diff
@@ -15,26 +15,19 @@ class Fish < Formula
     url "https://github.com/fish-shell/fish-shell.git", :shallow => false
 
     depends_on "autoconf" => :build
-    # Indeed, the head build always builds documentation
     depends_on "doxygen" => :build
   end
 
-  skip_clean "share/doc"
-
   def install
     system "autoconf" if build.head?
     # In Homebrew's 'superenv' sed's path will be incompatible, so
     # the correct path is passed into configure here.
     system "./configure", "--prefix=#{prefix}", "SED=/usr/bin/sed"
     system "make", "install"
-  end
 
-  def post_install
-    system "pkill fishd || true"
```

Comment:
[Yup](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/caveats.rb#L174-L187). Theoretically, if you wanted to add a `caveat` to the existing block with something like:

```
Before you run fish you must execute:
  `pkill fishd || true`
```

I'd be cool with that, at least.